library/src/main/java/com/novoda/downloadmanager/lib/Cursors.java

```diff
@@ -0,0 +1,27 @@
+package com.novoda.downloadmanager.lib;
+
+import android.database.Cursor;
+
+final class Cursors {
+
+    private Cursors() {
+        // no instances
+    }
+
+    public static int getInt(Cursor c, String column) {
+        return c.getInt(columnIndexFor(c, column));
+    }
+
+    public static long getLong(Cursor c, String column) {
+        return c.getLong(columnIndexFor(c, column));
+    }
+
+    public static String getString(Cursor c, String column) {
+        return c.getString(columnIndexFor(c, column));
+    }
+
+    private static int columnIndexFor(Cursor c, String column) {
+        return c.getColumnIndexOrThrow(column);
```

Comment:
not sure it's wise to have this `OrThrow` hidden inside a method that does not explicitly say that. When I first read line 12 I was wondering why you didn't use `OrThrow` but you do just that the method is named wrong. I would inline `columnIndexFor` or rename it